### PR TITLE
Add accessbility attributes

### PR DESCRIPTION
**Pull request by yiranli624**
## Is this a fix or a feature?
Fix

## What is the change?
n/a

## What does it fix? 
Add accessibility attributes such as language, aria-label, removing span, replace div with section

## Where should the reviewer start?
line 2 in index.html file

## How should this be tested?
run lighthouse on dev

### Screenshot (optional)

@atownse
@elle624
@lo-la-do-li

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -8,224 +8,223 @@
...
-        <div id='dropdown-friends-steps-container'></div>
+        <section id='dropdown-friends-steps-container'></section>
       </section>
     </header>
     <main>
       <section class='card-container' id='steps-card-container'>
         <section class='main-card' id='steps-main-card'>
-          <div class='main-card-top-row'>
-            <button type='button' name='button' class='info-button steps-info-button'></button>
+          <section class='main-card-top-row'>
```
Comment (atownse) [inline src/index.html:25]: Changed divs to sections, should be good enough semantic html for accessibility

File: src/index.html [hunk 2]
```diff
@@ -8,224 +8,223 @@
   </head>
   <body>
     <header>
-      <h1><span id='header-name'></span>FITLIT</h1>
-      <button type='button' name='button' id='profile-button'></button>
+      <h1 id='header-name'>FITLIT</h1>
+      <button aria-label='profile' id='profile-button'></button>
       <section class='hide' id='user-info-dropdown'>
         <h5 id='dropdown-name'></h5>
```
Comment (atownse) [inline src/index.html:14]: Find out how to deal with moving from h1 to h5 instead of h2 to improve accessibility

**Review by atownse — APPROVED** [review]

File: src/index.html (hunk 2 above)
Comment (yiranli624) [reply src/index.html:14]: Agreed! We will look at the CSS part once SASS is primarily set

Comment (lo-la-do-li) [reply src/index.html:14]: I agree, let's look at CSS and html together so we don't change any tags that have styling :) 
Well done adding all of the aria-labels. There are some instances where the class name is not really descriptive, and we could state the 'role' and use 'aria-labelledBy' attributes. 
For example:
```html
<section class='card-data-line' role="sleep-superlatives" aria-labelledBy=''sleep-friend-worst-sleeper">
 <p>LAST NIGHT'S INFERIOR SLEEPER</p>
 <h4 id='sleep-friend-worst-sleeper'></h4>
```

**Review by lo-la-do-li — APPROVED** [review]
We could potentially change h4 tags to p tags, but for now this looks like good use of aria-labeling for semantic html.

**Pull request merged**